tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/localedistance/LikelySubtagsBuilder.java

```diff
@@ -270,8 +270,10 @@ private static Map<String, Map<String, Map<String, LSR>>> makeTable(
 
         // Add the special case for "und-Latn" => "en-Latn-US" (which is a bit of a
         // hack for language matching).
-        // TODO: Find out the history of this line and document it better.
+        // Temporary patch. Needs an update of the ICU algorithm to match CLDR.
+        // See https://unicode-org.atlassian.net/browse/ICU-23052
```

Comment:
Remember that when you merge this PR for ICU-23052 in ICU 77, we have to close the ticket as fixed, and clone it for fixing the algorithm.

I suggest that you instead change this PR to use the BRS 77 ticket ICU-22923 and move 23052 to ICU 78.